src/roles/foreman/defaults/main.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 foreman_container_image: "quay.io/evgeni/foreman-rpm"
 foreman_container_tag: "nightly"
+foreman_listen_stream: localhost:3000
```

Comment:
I think what I don't know enough about is why we wouldn't default this to ` /run/httpd.foreman.sock` if our intent is to use systemd sock activation.